test/unit/messaging/schemas/remaining-amount.test.js

```diff
@@ -1,7 +1,6 @@
-const REMAINING_AMOUNT = require('../../../mocks/components/remaining-amount')
```

Comment:
Now we need to test out this schema against each of these valid test cases.

The describes on lines 39 and 65 need replicated for when remaining amount is top-up, when remaining amount is zero value, when remaining amount is... etc

OOS for this ticket but let's add a ticket in the backlog.

Reply:
Can shorten to 1 line

`const { topUpRemainingAmount: REMAINING_AMOUNT } = require('../../../mocks/components/remaining-amount')`

Reply:
The are is already a ticket for reviewing remainingAmount test files. I am sure we would be able to address auto test with Joi components and their usage.